float64 Xvalue
float64 Yvalue
float64 Zvalue
float64 dXvalue
float64 dYvalue
float64 dZvalue
float64 dYawvalue

int64   mode
uint8   grid=0
uint8   velocityEarth=1
uint8   velocityBody=2

int64   modeSwitch
int64   landed
int64   hover

int64   frame